src/Controls/tests/DeviceTests/Platforms/Windows/Package.appxmanifest

```diff
@@ -6,10 +6,7 @@
   xmlns:rescap="http://schemas.microsoft.com/appx/manifest/foundation/windows10/restrictedcapabilities"
   IgnorableNamespaces="uap rescap">
 
-  <Identity
-    Name="2C0C8B63-C6D1-48CC-A934-5108983E28EE"
-    Publisher="CN=User Name"
-    Version="1.0.0.0" />
+  <Identity Publisher="CN=User Name" />
```

Comment:
Not hardcoding this will make it pick up the application ID from the csproj through single project